source/_posts/2022-10-05-release-202210.markdown

```diff
@@ -383,15 +383,67 @@ for that specific item.
 
 {% details "Bayesian" %}
 
-The `prob_given_false` configuration variable is now a required parameter as there
-was no mathematical rationale for the previous default value.
-
-`numeric_state`, `template`, and `state` entries with only one `to_state`
-configured will update the prior probability accordingly if the
-observation is false. Those who have used duplicate, mirrored `state`
-configurations as a workaround will have their functionality preserved.
-However, for `numeric_state` and `template` entries, this will cause
-duplication of the Bayesian updating.
+As of 2022.10 `prob_given_false` is now a required configuration variable. Previously it had a default value of `1 - prob_given_true`, this didn't really have any mathematical basis and so has been removed. You can use this documentation to calculate what it should be, or to restore your previous functionality set it to `1 - prob_given_true`.
+
+In 2022.10 the logic of Bayesian sensors was significantly corrected. Previously the configs given in the examples above would not have worked as expected. For example if the device `device_tracker.paulus` was `home` this would not have updated the probability that `Paulus Home` was on! This is because Bayesian ignored observations that were false, even though that *should* update our probabilities. 
```

Comment:
```suggestion
The logic of Bayesian sensors has been significantly corrected. Previously the configs given in the examples above would not have worked as expected. For example, if the device `device_tracker.paulus` was `home`, this would not have updated the probability that `Paulus Home` was on. This is because Bayesian ignored observations that were false, even though that *should* update our probabilities. 
```